libs/design/notification/src/notification/notification.component.ts

```diff
@@ -55,10 +58,17 @@ export class DaffNotificationComponent
   implements DaffPrefixable, DaffStatusable {
   faTimes = faTimes;
 
+  @ContentChild(DaffPrefixDirective) _prefix: DaffPrefixDirective;
+
+  @ContentChild(DaffNotificationActionsDirective) _actions: DaffNotificationActionsDirective;
+
   @HostBinding('class.daff-notification') class = true;
 
   @HostBinding('attr.tabindex') tabindex = '0';
-  @HostBinding('attr.aria-live') ariaLive = 'polite';
+
+  @HostBinding('attr.role') get role() {
+    return this._actions || this.status === DaffStatusEnum.Warn || this.status === DaffStatusEnum.Danger ? 'alert' : 'status';
```

Comment:
Should `info` notifications with actions have status `alert`? I think this should just follow from the `status` prop.

Reply:
updated